### ISSUE TITLE
Support Apple Silicon M1/M2 (darwin arm64)

### ISSUE THREAD

**Issue by bajtos** [filecoin-saturn/L2-node#35]
We are already building darwin-arm64 version of saturn-l2. Early versions up to v0.0.3 worked well. Starting from the version v0.0.4, the darwin-arm64 version crashes at startup:

```
./saturn-l2
[1]    16654 killed     ./saturn-l2
```

This is probably required to enable Apple Silicon support for Filecoin Station, see https://github.com/filecoin-project/filecoin-station/issues/35

Possibly related threads:
- https://github.com/golang/go/issues/42684
- https://github.com/goreleaser/goreleaser/issues/1227
- https://goreleaser.com/customization/sign/?h=gon#with-gon

## TODO

- [x] Sign and notarize x86_64 https://github.com/filecoin-saturn/L2-node/pull/53
- [ ] Sign and notarize arm64 https://github.com/mitchellh/gon/issues/40#issuecomment-1258462989

**Comment by lidel** [comment]
@bajtos you'd have to make me Maintainer or Admin of this repo: I don't see the Settings tab atm.

While I am happy to help you with this, perhaps it is better if you do it  yourself: this is a good opportunity to remove a single point of failure from your release pipeline (my availability).

@patrickwoodhead  thoughts?  iiuc if anything happens to me and apple signing stops working, you will be in firefighting mode to figure things out.  I believe Saturn team should have Apple Developer accounts and access to signing-related secrets in the vault, so your team is not blocked on me.

**Comment by bajtos** [comment]
> While I am happy to help you with this, perhaps it is better if you do it yourself: this is a good opportunity to remove a single point of failure from your release pipeline (my availability).

That's a great idea!

@juliangruber will work on this.

**Comment by bajtos** [comment]
IIRC, we have already implemented the necessary changes and this GH issue can be closed as done.

@juliangruber feel free to open follow-up issue(s) for any improvements or bugs you are aware of.

**Comment by bajtos** [comment]
Ah, I was in a wrong repo 🙈 

I don't think anybody needs Saturn L2 on Apple Silicon, so let's close this as not planned.
